templates/hooks/parametergroup/delta_post_compare.go.tpl

```diff
@@ -0,0 +1,11 @@
+if delta.DifferentAt("Spec.ParameterGroupName") {
+    common.RemoveFromDelta(delta, "Spec.ParameterGroupName")
+}
+
+if delta.DifferentAt("Spec.ParameterGroupName") {
+    common.RemoveFromDelta(delta, "Spec.ParameterGroupName")
+}
```

Comment:
This is duplicated. But regardless, what is the purpose of this?

Reply:
Nice catch, Should have been `Spec.Family` only `ParameterNameValues` can be modified rest all are immutable